checker.py

```diff
@@ -239,17 +255,17 @@ def git_checker(
         report += 'The following directories ({}) have dirty WDs:\n{}'.format(
             len(dirty_dirs), reportify(dirty_dirs)
         )
-    if len(dirty_dirs) > 0 and len(unpushed_dirs) > 0:
+    if len(dirty_dirs) > 0 and len(unpushed_branches) > 0:
         report += '\n\n'
-    if len(unpushed_dirs) > 0:
-        report += 'The following directories ({}) need to be pushed:\n{}'.format(
-            len(unpushed_dirs), reportify(unpushed_dirs)
+    if len(unpushed_branches) > 0:
+        report += 'The following directories+branches ({}) need to be pushed:\n{}'.format(
```

Comment:
If we go with the above, this could be `...directories (+branches)` to show the branches appear optionally.